data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqssource/config/SqsSourceConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.sqssource.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SqsSourceConfig {
+
+    @JsonProperty("queues")
+    private QueuesOptions queues;
+
+    @JsonProperty("aws")
+    private AwsAuthenticationOptions aws;
```

Comment:
Add the `@Valid` annotation to these classes or their validation annotations will not run

Reply:
Thanks @graytaylor0 for your review, Added and available for your review.